components/link/link.js

```diff
@@ -115,6 +118,23 @@ class Link extends FocusMixin(LitElement) {
 					overflow: hidden;
 					overflow-wrap: anywhere;
 				}
+				d2l-icon.d2l-new-window {
+					color: var(--d2l-color-celestine);
+					height: 0.95em;
+					margin-inline-start: 0.315em;
+					vertical-align: inherit;
+					width: 0.95em;
+				}
+
+				a:hover d2l-icon.d2l-new-window {
+					--d2l-icon-fill-color: var(--d2l-color-celestine-minus-1);
+				}
+
+				@media print {
+					d2l-icon.d2l-new-window {
+						display: none;
+					}
+				}
```

Comment:
Now that this is conditionally rendered always, I think we can ditch the `d2l-new-window` CSS class?

Reply:
I added the class because when other icons were placed in the slot it would get colored blue (which may not be wanted), maybe now that it's part of the scoped styles that won't happen though?

Reply:
Tested this and it works fine now, other icons in the slot are still grey

Reply:
Ok excellent... yeah that would surprise me -- I'd only expect them to be affected if `::slotted(d2l-icon)` were used.